server/stack.yaml.lock

```diff
@@ -0,0 +1,103 @@
+# This file was autogenerated by Stack.
```

Comment:
`stack.yaml.lock` is introduced in `stack` build tool from `2.1.1`. It is recommended to put it in version control. Refer [here](https://docs.haskellstack.org/en/stable/lock_files/) to know more.